src/App.tsx

```diff
@@ -0,0 +1,200 @@
+                      height: theme.sizing.scale800,
+                      width: theme.sizing.scale800,
+                    })}
+                    viewBox="0 0 24 24"
+                  >
+                    <path
+                      fill="#aaaaaa"
+                      d="M17.65,6.35C16.2,4.9 14.21,4 12,4A8,8 0 0,0 4,12A8,8 0 0,0 12,20C15.73,20 18.84,17.45 19.73,14H17.65C16.83,16.33 14.61,18 12,18A6,6 0 0,1 6,12A6,6 0 0,1 12,6C13.66,6 15.14,6.69 16.22,7.78L13,11H20V4L17.65,6.35Z"
+                    />
+                  </svg>
+                </Button>
+              ),
+            }}
+          />
+        </StyledBody>
+
+        <StyledAction>
+          <Button
+            onClick={() => {
+              navigator.clipboard.writeText(password);
+              setCopied(true);
+            }}
+            overrides={{
+              BaseButton: {
+                style: {
+                  backgroundColor: "limegreen",
+                  width: "100%",
+                  marginBottom: "10px",
+                  ":hover": {
+                    backgroundColor: "green",
+                  },
+                },
+              },
+            }}
+            disabled={password === ""}
+          >
+            {copied ? "COPIED" : "COPY"}
+          </Button>
+        </StyledAction>
+        {/* #6943ff */}
+        <Accordion
+          accordion
+          overrides={{
+            Root: {
+              style: {
+                borderRadius: "1em",
+                overflow: "hidden",
+                backgroundColor: "#dbdbdb",
+              },
+            },
+            Content: {
+              style: {
+                backgroundColor: "#dbdbdb",
+              },
+            },
+            Header: {
+              style: {
+                borderRadius: "1em",
+                backgroundColor: "#dbdbdb",
+                color: "blue",
+              },
+            },
+          }}
+        >
+          <Panel title="Options">
+            <FormControl label="Length">
+              <Slider
+                value={length}
+                onChange={({ value }) => setLength(value)}
+                min={0}
+                max={20}
+              />
+            </FormControl>
+
+            <Checkbox
+              checked={capitalAllowed}
+              onChange={(e) => setCapitalAllowed(e.target.checked)}
+              labelPlacement={LABEL_PLACEMENT.right}
+            >
+              A-Z
+            </Checkbox>
+
+            <Checkbox
+              checked={numberAllowed}
+              onChange={(e) => setNumberAllowed(e.target.checked)}
+              labelPlacement={LABEL_PLACEMENT.right}
+            >
+              0-9
+            </Checkbox>
+            <Checkbox
+              checked={symbolAllowed}
+              onChange={(e) => setSymbolAllowed(e.target.checked)}
+              labelPlacement={LABEL_PLACEMENT.right}
+            >
+              %$#
+            </Checkbox>
```

Comment:
When you see some piece of code being repeated, try to extract it in a separate component. This way as the feature/requirement grows, you don't have to update it in `n` places

You can create a reusable component for Checkbox here. 

Following props can be passed - 
checked, onChange, children